models/review.go

```diff
@@ -558,6 +559,32 @@ func MarkReviewsAsNotStale(issueID int64, commitID string) (err error) {
 	return
 }
 
+// MarkReviewAsDismissed marks existing reviews as stale
+func MarkReviewAsDismissed(review *Review) (err error) {
```

Comment:
why not combine MarkReviewAsDismissed and MarkReviewAsUnDismissed?
since the only divverent line is `review.Dismissed = x` witch could be an function argument